src/core_types.jl

```diff
@@ -102,11 +102,11 @@ Base.@propagate_inbounds Base.getindex(r::RotMatrix, i::Int) = r.mat[i]
 @inline (::Type{RotMatrix{2,T}})(θ::Real) where {T} = RotMatrix(@SMatrix T[cos(θ) -sin(θ); sin(θ) cos(θ)])
 
 # A rotation is more-or-less defined as being an orthogonal (or unitary) matrix
-Base.inv(r::RotMatrix) = RotMatrix(r.mat')
+inv(r::RotMatrix) = RotMatrix(r.mat')
 
 # A useful constructor for identity rotation (eye is already provided by StaticArrays, but needs an eltype)
-@inline Base.eye(::Type{RotMatrix{N}}) where {N} = RotMatrix((eye(SMatrix{N,N,Float64})))
-@inline Base.eye(::Type{RotMatrix{N,T}}) where {N,T} = RotMatrix((eye(SMatrix{N,N,T})))
+@inline eye(::Type{RotMatrix{N}}) where {N} = RotMatrix((eye(SMatrix{N,N,Float64})))
```

Comment:
Ideally we'll add a deprecation warning, and maybe suggest using `one`.

We possibly need to support constructors like `RotXYZ(I)`,